sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -1,11 +1,29 @@
+- [Azure App Configuration client library for Python](#azure-app-configuration-client-library-for-python "Azure App Configuration client library for Python")
```

Comment:
Also, I completely missed this - why did you decide to move this section to the top? I don't think we need a menu. I'd just keep the examples (lines 11-14 here) listed (if anything at all) at the beginning of the examples section.